src/GafferUI/StandardNodeGadget.cpp

```diff
@@ -400,9 +400,24 @@ void StandardNodeGadget::doRenderLayer( Layer layer, const Style *style ) const
 				m_userColor.get_ptr()
 			);
 
-			if( MetadataAlgo::getBookmarked( node() ) )
+			int value = MetadataAlgo::getNumericBookmark( node() );
+			// \todo: should we support showing larger values?
```

Comment:
I think it makes more sense to enforce a 0-9 range in the API itself, so as well as throwing for `< 0`, we should throw for ` > 9`. Our intention is just to provide a small number of easily accessed bookmarks - after 9 they're no more easily accessed than a regular bookmark. And I know for sure I couldn't remember what I'd done with more than 10 of 'em at a time...